generate/schemagen/schemagen.go

```diff
@@ -25,7 +25,10 @@ import (
 )
 
 func main() {
-	var adminFacades = flag.Bool("admin-facades", false, "add the admin facades when generating the schema")
+	var (
+		adminFacades = flag.Bool("admin-facades", false, "add the admin facades when generating the schema")
+		facadeGroup  = flag.String("facade-group", "latest", "facade group to export (latest, all, client, jimm)")
```

Comment:
Could this take a list? `facade-groups` Saves us from running it twice and merging the output.

```
go run github.com/juju/juju/generate/schemagen -admin-facades --facade-groups=client,jimm ./apiserver/facades/schema.json
```

Reply:
Sure, was thinking about that...